kiwi/tasks/system_create.py

```diff
@@ -83,7 +83,12 @@ def process(self):
         )
 
         self.run_checks(
-            {'check_target_directory_not_in_shared_cache': [abs_root_path]}
+            {
```

Comment:
I guess we miss an equivalent check for the system_build task.

Reply:
yes, forgot to add it there. Thanks good catch and done